### PR TITLE
Notifications Design (Mobile)

### PR DESCRIPTION
**Pull request by koulopoulos**
# Purpose

- Implements the Twilio SMS notification registration flow designs for mobile.

# Tickets

- [Trello - Notifications Design (Mobile)](https://trello.com/c/SLqr6ozL)

# Contributors

- @koulopoulos 

# Feature List

###### A list of more in-depth and technical changes, additions, deletions, and fixes this PR provides.

# Notes

- Fixed toggle switch design for both desktop and mobile notifications

# Checklist

- [x] Filled out PR template :wink:
- [ ] Approved by designers
- [ ] Is passing linting checks
- [ ] Is passing tsc
- [ ] Is passing existing tests
- [ ] Has documentation and comments in code
- [ ] Has test coverage for code
- [ ] Will have clear squash commit message

<br>
@sandboxnu/searchneu

### CODE REVIEW

File: components/ResultsPage/Results/CourseResult.tsx [hunk 1]
```diff
@@ -280,14 +280,19 @@ export function MobileCourseResult({
                 course={course}
                 userInfo={userInfo}
                 onSignIn={onSignIn}
-                showNotificationSignup={false}
+                showNotificationSignup={true}
```
Comment (hankewyczz) [inline components/ResultsPage/Results/CourseResult.tsx]: remove feature flag? (and for desktop)

**Review by hankewyczz — APPROVED** [review]

File: styles/pages/_Results.scss [hunk 2]
```diff
@@ -204,6 +206,7 @@ $SIDEBAR_WIDTH: 268px;
   }
 
   &__dropdown {
+    z-index: 999999;
```
Comment (andrewydai) [inline styles/pages/_Results.scss:209]: 😭

**Review by andrewydai — APPROVED** [review]
🛸

File: styles/pages/_Results.scss (hunk 2 above)
Comment (koulopoulos) [reply styles/pages/_Results.scss:209]: 😎

File: components/ResultsPage/Results/CourseResult.tsx (hunk 1 above)
Comment (koulopoulos) [reply components/ResultsPage/Results/CourseResult.tsx]: Turns out I am an idiot. Resolved by [e9914f9](https://github.com/sandboxnu/searchneu/pull/163/commits/e9914f9076affbd351ef2e4f991b39c18dee15c3).

**Pull request merged**